game.p8

```diff
@@ -12,38 +12,38 @@ __lua__
-00000000eeeeeeeeeeeeeeeeeeeeeeee0aaaaa90a90a900e0aaaaa90a90a900eeeeeeeeeeeee09aaaaaaaaaaaaaaaaaaaa90eeee000000000000000000000000
-00000000eeeeeeeeeeeeeeeeeeeeeeeee0aaa90a90000eeee0aaa90a90000eeeeeeeeeeeeeee09aa0000000000000000aa90eeee000000000000000000000000
-00000000eeeeeeeeeeeeeeeeeeeeeeeee00a900905150eeee00a900905150eeeeeeeeeeeeeeee0009999999999999999000eeeee000000000000000000000000
-00000000eeeeeeeeeeeeeeeeeeeeeeeeee00000011b10eeeee00000011b10eeeeeeeeeeeeeee0999aaa000aaaaaaaaaa9990eeee000000000000000000000000
-00000000eeeeeeeeeeeeeeeeeeeeeeeee00b551115150eeee00b551115150eeeeeeeeeeeeee09aaaaa00001aaaaaaaaaaaa90eee000000000000000000000000
-00000000eeeeeeeeeeeeeeeeeeeeeeeeee0000000000eeeeee0000000000eeeeeeeeeeeeeee09aaaaa00001aaaaaaaaaaaa90eee000000000000000000000000
-00000000e000000ee000000eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee09aaaaa00001aaaaaaaaaaaa90eee000000000000000000000000
-000000000444444007777770eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee09aaaaa00011aaaaaaaaaaaa90eee000000000000000000000000
-000000000644446007788770eeeeeeeeeeee00000000eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee0999aaa111aaaaaaaaaa9990eeee000000000000000000000000
-0000000006a99a6007888870eeeeeeeeee00aaaaaaaa00eeeeeeeeeeeeeeeeeeeeeeeeeeeeeee0009999999999999999000eeeee000000000000000000000000
-0000000006a99a6007888870eeeeeeeee0aaaaaaaaaa990eeeeeeeeeeeeeeeeeeeeeeeeeeeee09aa0000000000000000aa90eeee000000000000000000000000
-0000000006aaaa6007788770eeeeeeee0d0aaaaaa9990010eeeeeeeeeeeeeeeeeeeeeeeeeeee09aaaaaaaaaaaaaaaaaaaa90eeee000000000000000000000000
-000000000666666007777770eeeeeeee0dd0aaaa9000d510eeeeeeeeeeeeeeeeeeeeeeeeeeee09aaaaaaaaaaaaaaaaaaaa90eeee000000000000000000000000
-00000000e000000ee000000eeeeeeeeee000aa990ddd000eeeeeeeeeeeeeeeeeeeeeeeeeeeee09999aaaaaaaaaaaaaa99990eeee000000000000000000000000
-00000000e000000ee000000eeeeeeeeee0d009900000dd0eeeeeeeeeeeeeeeeeeeeeeeeeeeeee0000999aaaaaaaa9990000eeeee000000000000000000000000
-000000000663366005555550eeeeeeee0ddd0990dddddd10eeeeeeeeeeeeeeeeeeeeeeeeeeeeee09a00099999999000a90eeeeee000000000000000000000000
-00000000063bb36005555550eeeeeeee0ddd090dddddd510eeeeeeeeeeeeeeeeeeeeeeeeeeeeee09aaaa00000000aaaa90eeeeee000000000000000000000000
-0000000003bbbb3005555550eeeeeeee0dddd0ddddddd510eeeeeeeeeeeeeeeeeeeeeeeeeeeeeee09aaaaaaaaaaaaaa90eeeeeee000000000000000000000000
-0000000003bbbb3005555550eeeeeeee0dddddddddd55110eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee099aaaaaaaaaa990eeeeeeee000000000000000000000000
-00000000063bb36005555550eeeeeeeee0dddddd5551110eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee00999999999900eeeeeeeee000000000000000000000000
-000000000663366005555550eeeeeeeeee001111111100eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee0000000000eeeeeeeeeee000000000000000000000000
-00000000e000000ee000000eeeeeeeeeeeee00000000eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee000000000000000000000000
+00000000ecdcecdcedcdedcdeeeeeeeeeee000eee000eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee0000000ee
```

Comment:
I added a temporary boss bee sprite here as I was trying to get it working.

Reply:
> (Not yet ready for review, btw)

Ah ok, lemme know when